### DISCUSSION TITLE
NanoEls build with Nema 34 motor on 250 x 550 lathe

### DISCUSSION THREAD

**Discussion by WillieJilesen** [kachurovskiy/nanoels#39] [Show and tell]
Hello

Here some pictures of my NanoEls build, this time on a 250 x 550 lathe.
It is also not a real NanoEls, because I have used a Arduino Uno together with a LCD dislay and other type of switches.
![IMG_1668](https://user-images.githubusercontent.com/96687625/147409138-abdbd74f-5ded-4524-a9a9-a3f61345c9a3.JPG)

The stepper motor is Nema 34 with 4.5 Nm . For the encoder , I used a 90 teeth gearwheel coupled to the 40 teeth gear standard installed.

The steper motor driver and power supply are attached to the wall, together with the frequency drive of the lathe

![IMG_1667](https://user-images.githubusercontent.com/96687625/147410028-095d4ee8-3f2d-45d1-ba95-f4db66d62bf5.JPG)

**Comment by kachurovskiy** [comment]
Willie, awesome build with lots of great ideas! Love how the encoder is mounted, especially the ability to adjust the gear meshing. Using one part for both motor mount and housing is also pretty cool :) Did you use a breadboard instead of a PCB? What are those push switches? Thank you.

**Comment by WillieJilesen** [comment]
Hello Maxim

First of all , thank you for providing all the information and Arduino software to make this possible. It was really well documented.
So well, that even I could build it, with very little experience in Arduino.

I used a screw terminal shield on the Arduino Uno, and then connected everything with wires. I hate soldering BTW.

For the pushbuttons, see attached picture. But with different colors . Here I had to solder anyway.

The dislay is the LCD type, but I think I will revert to the OLED type, because the LCD is not very clear and does not udate fast.

Regards from the Netherlands, Willie

Verzonden vanuit Mail voor Windows

Van: Maxim Kachurovskiy
Verzonden: zondag 26 december 2021 15:11
Aan: kachurovskiy/nanoels
CC: WillieJilesen; Author
Onderwerp: Re: [kachurovskiy/nanoels] NanoEls build with Nema 34 motor on 250 x550 lathe (Discussion #39)

Willie, awesome build with lots of great ideas! Love how the encoder is mounted, especially the ability to adjust the gear meshing. Using one part for both motor mount and housing is also pretty cool :) Did you use a breadboard instead of a PCB? What are those push switches? Thank you.
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by mockendon** [comment]
Nice!  Goes to show how much more torque you get from going with a wider stepper compared to s longer one.  I very much like the idea of VFD for the motor (having just rebuilt my pwm controlled following a diode failure that involved smd rework I’ll be stealing your idea next time it fails!).

**Comment by jozefz1979** [comment]
[Willie Jilesen] 
Excellent work, my friend. Can you share the modified code that you used to achieve the configuration of your design? Thanks.
